package.json

```diff
@@ -5,9 +5,10 @@
   "main": "index.js",
   "scripts": {
     "test": "karma start --single-run",
-    "dist": "mkdirp dist && browserify index.js -o dist/buildium-angular-ui.js",
+    "compile": "babel src --out-dir lib/ --presets=es2015",
+    "dist": "mkdirp dist && browserify lib/index.js -o dist/buildium-angular-ui.js",
     "docs": "node .build/docs.js",
-    "prepublish": "npm run dist",
+    "prepublish": "npm run compile && npm run dist",
     "version": "npm run docs -- --gh-pages"
```

Comment:
right, it's not needed, it's more of a convenience so we can build the package without publishing (and also not overloading the use of _prepublish_)